datastore/models.go

```diff
@@ -1312,17 +1324,45 @@ type Organisation struct {
 }
 
 type Configuration struct {
-	UID                string                        `json:"uid" db:"id"`
-	IsAnalyticsEnabled bool                          `json:"is_analytics_enabled" db:"is_analytics_enabled"`
-	IsSignupEnabled    bool                          `json:"is_signup_enabled" db:"is_signup_enabled"`
-	StoragePolicy      *StoragePolicyConfiguration   `json:"storage_policy" db:"storage_policy"`
-	RetentionPolicy    *RetentionPolicyConfiguration `json:"retention_policy" db:"retention_policy"`
+	UID                string `json:"uid" db:"id"`
+	IsAnalyticsEnabled bool   `json:"is_analytics_enabled" db:"is_analytics_enabled"`
+	IsSignupEnabled    bool   `json:"is_signup_enabled" db:"is_signup_enabled"`
+
+	StoragePolicy        *StoragePolicyConfiguration   `json:"storage_policy" db:"storage_policy"`
+	RetentionPolicy      *RetentionPolicyConfiguration `json:"retention_policy" db:"retention_policy"`
+	CircuitBreakerConfig *CircuitBreakerConfig         `json:"breaker_config" db:"breaker_config"`
```

Comment:
The name `breaker_config` is not consistent with `circuit_breaker` in config.go

Reply:
True, I'll update it